From 9032a185c42028c686d2eadfdca30b9fbd9bb984 Mon Sep 17 00:00:00 2001
From: Ranjani Sridharan <ranjani.sridharan@linux.intel.com>
Date: Tue, 12 Jun 2018 22:14:44 -0700
Subject: [PATCH 1689/2345] ASoC: uapi: sof: remove DMAC ID and DMAC channel
 tokens

No need to pass dmac id and dmac chan from topology as the firmware
does not use this info anymore.

Signed-off-by: Ranjani Sridharan <ranjani.sridharan@linux.intel.com>
---
 include/uapi/sound/sof-topology.h | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/include/uapi/sound/sof-topology.h b/include/uapi/sound/sof-topology.h
index 5941c4c..1a4957b 100644
--- a/include/uapi/sound/sof-topology.h
+++ b/include/uapi/sound/sof-topology.h
@@ -34,8 +34,6 @@
 #define SOF_TKN_BUF_CAPS			101
 
 /* DAI */
-#define SOF_TKN_DAI_DMAC			151
-#define	SOF_TKN_DAI_DMAC_CHAN			152
 #define	SOF_TKN_DAI_DMAC_CONFIG			153
 #define SOF_TKN_DAI_TYPE			154
 #define SOF_TKN_DAI_INDEX			155
@@ -58,8 +56,6 @@
 #define SOF_TKN_SRC_RATE_OUT			301
 
 /* PCM */
-#define SOF_TKN_PCM_DMAC			351
-#define SOF_TKN_PCM_DMAC_CHAN			352
 #define SOF_TKN_PCM_DMAC_CONFIG			353
 
 /* Generic components */
-- 
2.7.4

